src/backend/aspen/api/policy.polar

```diff
@@ -76,6 +76,9 @@ resource PhyloTree {
   "write" if "member";
 }
 
+has_permission(ac: AuthContext, "invite_members", group: Group) if
+    has_role(ac, "admin", group) or ac.user.system_admin;
```

Comment:
Support letting system admins send invitations (we basically need this to invite the first user to a brand new group)